codice/alliance
# CAL-48 Refactor alliance-imaging to use imaging-nitf 0.3

#### What does this PR do?
Updated class and method names as required by imaging-nitf 0.3.  Also changed the nitf-transformer to use the imaginag-nitf fluent API.

#### Who is reviewing it?
@bdeining 
@jlcsmith 
@jaymcnallie 

#### How should this be tested?
Ingest a NITF as you would have prior to this change.  The only difference in functionality is that I removed the 'nitf.numberOfImages' attribute.  That attribute won't be part of our taxonomy and it's easy enough to see based on the multi-valued fields.  The value is not easily retrievable in 0.3.

#### Any background context you want to provide?

#### What are the relevant tickets?
CAL-48

#### Screenshots (if appropriate)

#### Checklist:

No changes to documentation or tests.
